index.js

```diff
@@ -209,14 +209,16 @@ module.exports = class Cutecord extends Plugin {
       }
     }
 
+    const showLurk = this.settings.get('lurkedGuilds')
     const guildID = channel.getGuildId()
-    if (guildID && isLurking(guildID)) {
-      // Are we lurking the guild? Apparently we don't want notifications for that.
+    if (guildID && isLurking(guildID) && !showLurk) {
+      // Are we lurking the guild? Apparently we don't want notifications for that. Unless we do want them
       return false
     }
 
-    if (notificationSettings.allowNoMessages(channel)) {
-      // No.
+    const overwriteMuteSupression = this.settings.get('overwriteMuteSupression')
+    if (notificationSettings.allowNoMessages(channel) && !overwriteMuteSupression) {
+      // No. Unless we want them... I really do, don't judge me.
       return false
     }
```

Comment:
This logic doesn't match your changelog note, instead it makes all notifications from muted channels go through (not just those that match the cute list).

Reply:
Yes, this code does indeed let all muted notifications go through if the user decides to turn the feature on. However, later on in the method it would still need to pass a "cute" in order to return true. If it doesn't contain a cute it would check for DND and then finally drop it.

I've kept this check in place, but added an additional check after checking for cute, if there where no cute found and the message is muted, return that we don't want to get notified